server/src/main/java/io/crate/protocols/ssl/SslSettings.java

```diff
@@ -69,8 +69,11 @@ private SslSettings() {
 
         static SSLMode parse(String value) {
             return switch (value.toLowerCase(Locale.ENGLISH)) {
+                // allow true/false as well because YAML `on` is interpreted as true
                 case "on" -> ON;
+                case "true" -> ON;
                 case "off" -> OFF;
+                case "false" -> OFF;
```

Comment:
shouldn't this parse method be also used at https://github.com/crate/crate/pull/11350/files#diff-1a801fd9101cdf86ecf02380887786c93c57bde1c650e03c5842d02ce20b4f34R363?

Reply:
Somehow the link doesn't point me to a specific place?

Reply:
ah sorry, should link to `Netty4Transport.initChannel(...)`

Reply:
This?

https://github.com/crate/crate/blob/a039e1ff335fdc437da85be593280613eee9a3f3/server/src/main/java/org/elasticsearch/transport/netty4/Netty4Transport.java#L313-L314

It's already returning the parsed enum value

Reply:
ah ok, seems like I understood the usage or `parse` wrongly.